bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/thing/intellichem.xml

```diff
@@ -156,6 +176,10 @@
 		<item-type>Number:Time</item-type>
 		<label>Dose Time</label>
 		<description>The time a particular chemical has been dosing.</description>
+		<tags>
+			<tag>Control</tag>
```

Comment:
if its readonly, is it a control?  Maybe Status would be better?

Reply:
Changed to status